test-yolo.py

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env python
+# coding=utf-8
+
+#
+# Usage:
+#
+# Run all tests against PHP 7.3:
+#
+#   ./test-yolo.py ci-drupal-8
```

Comment:
The script is named test-yolo; have you moved all of the test fixtures to the yolo server?

Reply:
I have not. How would I do that?